src/query-helpers.js

```diff
@@ -64,6 +74,19 @@ function makeGetAllQuery(allQuery, getMissingError) {
         container,
       )
     }
+    if (getConfig().showSuggestions) {
+      const rawSuggestions = els.map(element => getSuggestedQuery({element}))
+      const suggestions = [
+        ...new Set(els.map(element => getSuggestedQuery({element}).toString())),
+      ]
+
+      if (
+        suggestions.length === 1 &&
+        !allQuery.name.endsWith(rawSuggestions[0].queryName)
+      ) {
+        throw getSuggestionError(suggestions[0], container)
```

Comment:
@kentcdodds  was pairing on this with my friend Kevin and he asked a good question.  Should the suggestion Try to actually execute the query it's suggesting just to make sure it works?  Curious to hear your thoughts.
..